### DISCUSSION TITLE
独立开发者出海周刊（第31期） | 信息差——独立开发者出海周刊

### DISCUSSION THREAD

**Discussion by giscus** [weijunext/gapis.money#82] [General]
# 独立开发者出海周刊（第31期） | 信息差——独立开发者出海周刊

Knowledge is power, info-gap is money!「信息差——独立开发者出海周刊」是一个帮助独立开发者缩小信息差的技术周刊。

https://gapis.money/weekly/2024-12-23_031

**Comment by WoJiaoFuXiaoYun** [comment]
周刊怎么停更了呀
